homeassistant/components/tessie/cover.py

```diff
@@ -105,3 +109,52 @@ async def async_close_cover(self, **kwargs: Any) -> None:
         """Close windows."""
         await self.run(close_charge_port)
         self.set((self.key, False))
+
+
+class TessieFrontTrunkEntity(TessieEntity, CoverEntity):
+    """Cover entity for the charge port."""
+
+    _attr_device_class = CoverDeviceClass.DOOR
+    _attr_supported_features = CoverEntityFeature.OPEN
+
+    def __init__(self, coordinator: TessieStateUpdateCoordinator) -> None:
+        """Initialize the sensor."""
+        super().__init__(coordinator, "vehicle_state_ft")
+
+    @property
+    def is_closed(self) -> bool | None:
+        """Return if the cover is closed or not."""
+        return self._value == 0
```

Comment:
I'd make 0 and 1 named constants so open and closed don't get accidentally confused in future refactoring some years down the road

Reply:
Good Idea, Done.